### PR TITLE
remove ctx and ns subcommand from colorization

### PR DESCRIPTION
**Pull request by prune998**
## WHAT
Remove colorization from `kubecolor ctx` and `kubecolor ns`

## WHY
`kubectl ctx` and `kubectl ns` returns a list colored white with one selected item in yellow.
The `ctx` and `ns` commands called from `kubecolor` return everything in yellow, making it impossible to find which context or namespace is currently used as default.

## Related issue (if exists)
https://github.com/hidetatz/kubecolor/issues/85

### CODE REVIEW

**Comment by prune998** [comment]
@hidetatz any news ? 
thx !

**Comment by prune998** [comment]
Anyone to review this ? @hidetatz  ?
Is this project still active ? Need new maintainers ?

**Comment by sontek** [comment]
What is holding this back?

**Comment by prune998** [comment]
The fact that the maintainer is not answering to any way I tried to contact ?

**Review by hidetatz — APPROVED** [review]
Cool, thanks

**Pull request merged**
